src/memgraph.cpp

```diff
@@ -756,43 +756,56 @@ class AuthQueryHandler final : public memgraph::query::AuthQueryHandler {
+        });
   }
 
   void DenyPrivilege(const std::string &user_or_role,
                      const std::vector<memgraph::query::AuthQuery::Privilege> &privileges,
-                     const std::vector<std::string> &labels, const std::vector<std::string> &edgeTypes) override {
-    EditPermissions(user_or_role, privileges, labels, edgeTypes, [](auto *permissions, const auto &permission) {
-      // TODO (mferencevic): should we first check that the
-      // privilege is granted/denied/revoked before
-      // unconditionally granting/denying/revoking it?
-      permissions->Deny(permission);
-    });
+                     const std::vector<std::string> &labels, const std::vector<std::string> &edge_types) override {
+    EditPermissions(
+        user_or_role, privileges, labels, edge_types,
+        [](auto &permissions, const auto &permission) {
+          // TODO (mferencevic): should we first check that the
+          // privilege is granted/denied/revoked before
+          // unconditionally granting/denying/revoking it?
+          permissions.Deny(permission);
+        },
+        [](auto &label_permissions, const auto &label) {
+          label_permissions.Deny(label, memgraph::auth::LabelPermission::READ);
+        });
   }
 
   void RevokePrivilege(const std::string &user_or_role,
                        const std::vector<memgraph::query::AuthQuery::Privilege> &privileges,
-                       const std::vector<std::string> &labels, const std::vector<std::string> &edgeTypes) override {
-    EditPermissions(user_or_role, privileges, labels, edgeTypes, [](auto *permissions, const auto &permission) {
-      // TODO (mferencevic): should we first check that the
-      // privilege is granted/denied/revoked before
-      // unconditionally granting/denying/revoking it?
-      permissions->Revoke(permission);
-    });
+                       const std::vector<std::string> &labels, const std::vector<std::string> &edge_types) override {
+    EditPermissions(
+        user_or_role, privileges, labels, edge_types,
+        [](auto &permissions, const auto &permission) {
+          // TODO (mferencevic): should we first check that the
+          // privilege is granted/denied/revoked before
+          // unconditionally granting/denying/revoking it?
+          permissions.Revoke(permission);
+        },
+        [](auto &label_permissions, const auto &label) { label_permissions.Revoke(label); });
   }
 
  private:
-  template <class TEditFun>
+  template <class TEditFun, class TEditLabelPermisionsFun>
   void EditPermissions(const std::string &user_or_role,
                        const std::vector<memgraph::query::AuthQuery::Privilege> &privileges,
                        const std::vector<std::string> &labels, const std::vector<std::string> &edgeTypes,
-                       const TEditFun &edit_fun) {
+                       const TEditFun &edit_fun, const TEditLabelPermisionsFun &edit_label_permisions_fun) {
```

Comment:
The typo is also below
```suggestion
                       const TEditFun &edit_fun, const TEditLabelPermisionsFun &edit_label_permissions_fun) {
```